Database/Table Scripts/CreatePetTableScript.sql

```diff
@@ -0,0 +1,14 @@
+CREATE TABLE dbo.Pet(
+	Id BIGINT PRIMARY KEY IDENTITY(1,1) NOT NULL,
+	OwnerId BIGINT NOT NULL FOREIGN KEY REFERENCES dbo.Client(Id),
+	Name VARCHAR(50) NOT NULL,
+	Breed VARCHAR(50) NOT NULL,
+	Type VARCHAR(50) NOT NULL,
+	Weight DECIMAL NOT NULL,
+	DOB VARCHAR(25) NOT NULL,
+	BordetellaVax BIT DEFAULT 0 NOT NULL,
+	DHPPVax BIT DEFAULT 0 NOT NULL, 
+	RabieVax BIT DEFAULT 0 NOT NULL, 
+	OtherInfo VARCHAR(2000) NOT NULL,
```

Comment:
Other info can be null. They are not required to put other information down if they don't want.